helm.rst

```diff
@@ -0,0 +1,161 @@
+Helm Charts
+===========
+
+The packaging of the CSC deployment starts with a `Helm <https://v2.helm.sh/>`_ chart. We using version 2 of Helm due the current ArgoCD restriction on the Helm version. The code for the charts are kept in the `Helm chart Github repository <https://github.com/lsst-ts/charts>`_. The next two sections will discuss each chart in detail. For a description of the APIs used, consult the `Kubernetes documentation <https://kubernetes.io/docs/reference/>`_ for the API reference. The chart sections will not go into great detail on the content of each API delivered. Each chart section will list all of the possible configuration aspects that each chart is delivering, but full use of that configuration is left to the `ArgoCD Configuration` section. For the CSC deployment, we will run a single container per pod on Kubernetes. The Kafka producers may follow a similar pattern, but they are allowed to scale beyond the single pod?
```

Comment:
Is "packaging" the correct word here? 

A couple of typos:

> We using -> We *are* using

When you mention that we are using v2 of Helm I think it would be a good idea to also list the version of argoCD. I would also rephrase that a little bit. A suggestion:

> We are currently adopting version X.X of ArgoCD which works with Helm version 2.

I think you need double `_` for inline links (https://developer.lsst.io/restructuredtext/style.html#external-links). 

API is repeated

> For a description of the APIs used, consult the `Kubernetes documentation <https://kubernetes.io/docs/reference/>`_ ~for the API reference~.

Is this a question? Not sure I understood what you are trying to say here. Can you clarify?

> The Kafka producers may follow a similar pattern, but they are allowed to scale beyond the single pod?

Reply:
The question: The Kafka producers may follow a similar pattern, but they are allowed to scale beyond the single pod? was actually meant for you since you understand the producers better. :-)

Reply:
Hmm. The links work as advertised for me. What behavior did you see?